test/bdd/steps/endpoints.js

```diff
@@ -541,6 +541,19 @@ Given(/^(DC|DV|DV2) waits for purchase to finish$/, { timeout: 300000 }, async f
 });
 
 
+Then(/^The last export doesn't have permissioned data$/, async function () {
+    expect(!!this.state.lastExport, 'Last export data not defined. Use other step to define it.').to.be.equal(true);
+    expect(this.state.lastExport.status, 'Last export should be completed.').to.be.equal('COMPLETED');
+    const { data } = this.state.lastExport;
+    const properties = (JSON.parse(data.formatted_dataset)['@graph']).map(x => x.properties);
+    for (const p of properties) {
+        if (p.permissioned_data) {
+            expect(p.permissioned_data).to.have.keys(['permissioned_data_hash']);
```

Comment:
This should also have a line with `to.not.have.keys(['data'])`

Reply:
I think that the first line is enough, but let's add one more check